kratos/python_scripts/run_tests.py

```diff
@@ -177,7 +177,7 @@ def print_summary(exit_codes):
 
 def main():
     # Define the command
-    cmd = os.path.join(os.path.dirname(KtsUtls.GetKratosMultiphysicsPath()), 'runkratos')
+    cmd = sys.executable
```

Comment:
Please use the fct from testing.utilities

Reply:
mm I've been looking for it but There seems to be no function to do this, which one are you suggestion?

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/9ecfcecde03ff2c5fcf05040a1efadb505c8b8bc/kratos/python_scripts/testing/utilities.py#L9-L14

Reply:
aaa thx!